lrrbot/rpc.py

```diff
@@ -98,8 +99,7 @@ async def get_header_info(self):
 				"is_override": self.lrrbot.show_override is not None,
 			}
 
-		if 'advice' in storage.data['responses']:
-			data['advice'] = random.choice(storage.data['responses']['advice']['response'])
+		data['advice'] = get_advice()
```

Comment:
The `bot` argument is missing.
```suggestion
		data['advice'] = get_advice(self.lrrbot)
```